### ISSUE TITLE
Can't access any of the BetterDiscord settings tabs.

### ISSUE THREAD

**Issue by GenesisAria** [rauenzi/BetterDiscordApp#80]
When i click on them i get nowhere... I tried installing once, it was working fine, then i added some plugins and suddenly it broke... Since then i tried reinstall entirely fresh discord with roaming betterdiscord folder completely cleared, but still son't  work.  Settings panel mostly ignores theme as well, this was not the case before.

![](https://i.imgur.com/bevMi0I.png)

**Comment by MoofMonkey** [comment]
+1

**Comment by GucciBoiFuccBoi** [comment]
![download](https://user-images.githubusercontent.com/44310358/47251036-122d2280-d3fb-11e8-9a24-de5a873f85a0.png)
I am also having the same issue but none of the tabs under the BBD header are responding/opening when I click them.

**Comment by SoWhoYou** [comment]
+1

**Comment by MoofMonkey** [comment]
> ![download](https://user-images.githubusercontent.com/44310358/47251036-122d2280-d3fb-11e8-9a24-de5a873f85a0.png)
> I am also having the same issue but none of the tabs under the BBD header are responding/opening when I click them.

We have the same problem.

**Comment by SaltyKiwi4303** [comment]
> 
> 
> > ![download](https://user-images.githubusercontent.com/44310358/47251036-122d2280-d3fb-11e8-9a24-de5a873f85a0.png)
> > I am also having the same issue but none of the tabs under the BBD header are responding/opening when I click them.
> 
> We have the same problem.

Yeah, same issue here :/

**Comment by Yohji52** [comment]
Same issue over here.

**Comment by Michael3038** [comment]
Yeah, lets make this a high priority issue.

**Comment by SmugCardboardBox** [comment]
![screenshot_87](https://user-images.githubusercontent.com/32379718/47251319-2d426700-d3e8-11e8-9d7c-a5fc0a5cf45a.png)


Same issue as well, got it to install fine, but none of the BetterDiscord tabs work

**Comment by XenHat** [comment]
Same here, installed fine, albeit with no BD logo in the corner (if that's supposed to still exist), and the settings tab labels are entirely non-functional

**Comment by StarWolf3000** [comment]
I have this problem too. I checked the Console and this message shows up, when you click on one of the BD entries: `FAILED TO LOCATE ROOT: .layer .ui-standard-sidebar-view`

**Comment by VegetaJunior** [comment]
> 
> 
> > > ![download](https://user-images.githubusercontent.com/44310358/47251036-122d2280-d3fb-11e8-9a24-de5a873f85a0.png)
> > > I am also having the same issue but none of the tabs under the BBD header are responding/opening when I click them.
> > 
> > 
> > We have the same problem.
> 
> Yeah, same issue here :/

Yeah, I tried uninstalling discord completely and reinstalling regular discord and then better discord and it's still the same issue.

**Comment by intrnl** [comment]
There's no need to +1 the issue, it is [known](https://discordapp.com/channels/292141134614888448/305865219412983808/503054705052745729).

Zerebos is busy catching up with real life, just wait.
